daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1493,39 +1495,26 @@ private[lf] object SBuiltin {
     // Callback from the engine returned NotFound
     def handleKeyFound(machine: Machine, cid: V.ContractId): Control
     // We already saw this key, but it was undefined or was archived
-    def handleKeyNotFound(machine: Machine, gkey: GlobalKey): (Control, Boolean)
-
-    final def handleKnownInputKey(
-        machine: Machine,
-        gkey: GlobalKey,
-        keyMapping: ContractStateMachine.KeyMapping,
-    ): Control =
-      keyMapping match {
-        case ContractStateMachine.KeyActive(cid) =>
-          handleKeyFound(machine, cid)
-        case ContractStateMachine.KeyInactive =>
-          val (control, _) = handleKeyNotFound(machine, gkey)
-          control
-      }
```

Comment:
the only use of `handleKnownInputKey` happened right after a successful match of `keyMapping` to `ContractStateMachine.KeyInactive`, so effectively it was the same as just calling `handleKeyNotFound`